odoo/tests/common.py

```diff
@@ -708,6 +708,7 @@ def profile(self, description='', **kwargs):
 savepoint_seq = itertools.count()
 
 
+mute_cache_logger = mute_logger(odoo.modules.registry._logger.name)
 class TransactionCase(BaseCase):
```

Comment:
My only comment, for the sake of readability: leave blank lines before the `class` keyword.
```suggestion
savepoint_seq = itertools.count()
mute_cache_logger = mute_logger(odoo.modules.registry._logger.name)


class TransactionCase(BaseCase):
```